### PR TITLE
#55/UI/main button

### PR DESCRIPTION
**Pull request by RobertS-ki**
obraz

obraz
This is an atom of button. Added third button variant, task was renamed to Main Button. Button variants are available through props "primary", "secondary", "ternary" - colors and styles adopted accordingly to props.
onClick handlers to be defined on the page that returns the buttons (on the picture rendered on excersizes page).

Styles use imports from FONTS and COLORS variable js files:
obraz

### CODE REVIEW

File: src/Pages/LoginPage/LoginPage.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import 'font-awesome/css/font-awesome.min.css';
 import InputField from '../../Atoms/InputField/InputField';
+<<<<<<< HEAD
```
Comment (MonikaKrella) [inline src/Pages/LoginPage/LoginPage.js:3]: it looks like lines with ====== or <<<<<<< or >>>>>> stayed after resolving conflicts

File: src/Atoms/MainBtn/MainBtn.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      {text}
+    </MainBtnStyled>
+  );
+};
+
+MainBtn.propTypes = {
+  primary: propTypes.string,
+  secondary: propTypes.string,
+  ternary: propTypes.string,
+  text: propTypes.string,
```
Comment (MonikaKrella) [inline src/Atoms/MainBtn/MainBtn.js:24]: shouldn't be text as required property? at figma, every button has text on button

File: src/Pages/ExercisesPage/ExercisesPage.js [hunk 3]
```diff
@@ -1,3 +1,27 @@
+<<<<<<< HEAD
```
Comment (MonikaKrella) [inline src/Pages/ExercisesPage/ExercisesPage.js:1]: here also it looks like something after resolving conflicts left (<<<<<<<. =======)

File: src/Atoms/MainBtn/MainBtnStyled.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  flex-direction: column;
+  align-items: center;
+`;
+
+const MainBtnStyled = styled.button`
+  display: block;
+  width: 343px;
+  height: 50px;
+  color: ${(props) =>
+    !props.primary
```
Comment (MonikaKrella) [inline src/Atoms/MainBtn/MainBtnStyled.js:16]: it smells a little bit like something for Themes in Styled components? But not in that issue, just as TODO - think in a team about themes

**Review by MonikaKrella — CHANGES_REQUESTED** [review]
errrors:
image

File: src/Atoms/MainBtn/MainBtnStyled.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      ? !props.secondary
+        ? `${COLORS.grey800}`
+        : `${COLORS.primary601}`
```
Comment (vieraboschkova) [inline src/Atoms/MainBtn/MainBtnStyled.js:19]: This becomes very hard to read and follow. If possible, try not to nest ternary operations.

File: src/Pages/LoginPage/LoginPage.js (hunk 1 above)
Comment (RobertS-ki) [reply src/Pages/LoginPage/LoginPage.js]: Fixed. Stange...

File: src/Atoms/MainBtn/MainBtn.js (hunk 2 above)
Comment (RobertS-ki) [reply src/Atoms/MainBtn/MainBtn.js]: Added isRequired.

File: src/Pages/ExercisesPage/ExercisesPage.js (hunk 3 above)
Comment (RobertS-ki) [reply src/Pages/ExercisesPage/ExercisesPage.js]: Fixed. Strange.

File: src/Atoms/MainBtn/MainBtnStyled.js (hunk 4 above)
Comment (RobertS-ki) [reply src/Atoms/MainBtn/MainBtnStyled.js]: TODO.

File: src/Atoms/MainBtn/MainBtnStyled.js (hunk 5 above)
Comment (RobertS-ki) [reply src/Atoms/MainBtn/MainBtnStyled.js]: Done.  One level ternary operators.

File: src/Atoms/MainBtn/MainBtnStyled.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  color: ${(props) =>
+    props.primary
+      ? COLORS.white
+      : props.secondary
+      ? COLORS.primary601
+      : COLORS.grey800};
+  ${FONTS.button};
+  background: ${(props) =>
+    props.primary
+      ? COLORS.primary501
```
Comment (vieraboschkova) [inline src/Atoms/MainBtn/MainBtnStyled.js:23]: That's not what I meant actually. I mean ternary operators are simple YES:NO solution, they should be used as that. If you have to use: YES: NO? YES: NO, its not a way to go. See if you can solve it in another way, withuot nesting the expression.

**Review by Smyku6 — APPROVED** [review]
Good Job Rob!

**Review by MonikaKrella — APPROVED** [review]

**Pull request merged**